tests/integration/helpers.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+from pathlib import Path
+import requests
+
+
+def start_docker_compose(file_path):
+    os.system(f"docker-compose -f {file_path} --project-directory . up  --build -d")
+
+
+def stop_docker_compose(file_path):
+    os.system(f"docker-compose -f {file_path} --project-directory . down")
+
+
+def call_api(method, url, payload=None, headers={"Content-Type": "application/json"}):
+    return getattr(requests, method)(
+        url, data=json.dumps(payload), headers=headers
+    ).json()
+
+
+def get_results(query, top_k=10):
+    return call_api(
+        method="post",
```

Comment:
use single quotes

Reply:
Double quotes are coming from the linter